indico/modules/events/timetable/controllers/legacy.py

```diff
@@ -103,6 +107,77 @@ def _process(self):
         return jsonify_form(form, fields=form._display_fields)
 
 
+class RHLegacyTimetableEditEntry(RHManageTimetableBase):
+    def _checkParams(self, params):
+        RHManageTimetableBase._checkParams(self, params)
+        self.timetable_entry = (self.event_new.timetable_entries
+                                .filter_by(id=request.view_args['timetable_entry_id'])
+                                .first_or_404())
+
+    def _process(self):
+        form = None
+        if self.timetable_entry.contribution:
+            contrib = self.timetable_entry.contribution
+            tt_entry_dt = self.timetable_entry.start_dt.astimezone(self.event_new.tzinfo)
+            form = ContributionEntryForm(obj=FormDefaults(contrib, time=tt_entry_dt.time()),
+                                         event=self.event_new, contrib=contrib, to_schedule=False,
+                                         day=tt_entry_dt.date())
+            if form.validate_on_submit():
+                # TODO: Check whether track_time_changes should be used.
```

Comment:
Yeah but shouldn't I first check if the start_dt/duration have changed beforehand? Or just use the context manager in any case (when start_dt and/or duration are included in the form)?